hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/rgromov/hw-golang-otus/tree/master/hw01_hello_otus
```

```suggestion
module github.com/rgromov/hw-golang-otus/hw01_hello_otus
```

`tree/master/` - относится к  UI-роутингу GitHub

---

Да-точно-спасибо!